lib/house.rb

```diff
@@ -0,0 +1,74 @@
+require 'pry'
+require './lib/room'
+class House
+  attr_reader :price,
+              :address,
+              :rooms
+
+  def initialize(price, address)
+    @price = price.delete("$").to_i
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room)
+    rooms << room
+  end
+
+  def above_market_value?
```

Comment:
Keep an eye on the interaction pattern!  This method works, but should be named `above_market_average?`